test/SmallTests/test_small.py

```diff
@@ -0,0 +1,50 @@
+import sys
+sys.path.append('../../')
+from ProjectLib import XMLtoProject
+from TimeLoopLib import TreeDynamicTimeStepping
+import unittest
+import glob
+import os
+
+filepathExampleSetups = "./Test_Setups_small/*.xml"
```

Comment:
This fails, if the file is run from another path.
```suggestion
        filepathExampleSetups = path.join(path.dirname(path.abspath(__file__)),"Test_Setups_small/*.xml")
```